pkg/appStore/deployment/tool/gitops/AppStoreDeploymentArgoCdService.go

```diff
@@ -463,3 +473,27 @@ func (impl AppStoreDeploymentArgoCdServiceImpl) updateValuesYaml(environment *cl
 	installAppVersionRequest.GitHash = commitHash
 	return installAppVersionRequest, nil
 }
+
+func (impl AppStoreDeploymentArgoCdServiceImpl) ParseGitRepoErrorResponse(err error) (bool, error) {
+	//update values yaml in chart
+	noTargetFound := false
+	if err != nil {
+		if errorResponse, ok := err.(*github.ErrorResponse); ok && errorResponse.Response.StatusCode == http.StatusNotFound {
+			impl.Logger.Errorw("no content found while updating git repo on github, do auto fix", "error", err)
+			noTargetFound = true
+		}
+		if errorResponse, ok := err.(azuredevops.WrappedError); ok && *errorResponse.StatusCode == http.StatusNotFound {
```

Comment:
azure sometimes also sends pointe to this wrapped error. please handle that case too.